tools/build-launchui.js

```diff
@@ -54,3 +55,13 @@ function buildLaunchUI() {
   if ( result.error != null )
     throw result.error;
 }
+
+function symlinkIfNotExist( src, dest ) {
```

Comment:
Old check was not enough and doesn't work. Now you can run `npm run build:libui` several times.